### PR TITLE
Release 4.4.0

### PR DESCRIPTION
**Pull request by daniloaliberti**
# Updates

- App Record - in AppStoreConnect - localised notes 
- Timings
- Delta time for issues tip

### CODE REVIEW

**Review by JasonDobo — APPROVED** [review]

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ There are usually two release engineers working at any given time. It goes witho
 *It starts after all opened issues had been adressed and can take several cycles until QA's approval*
 
 1. Triger a new release build in the `#ios-build` channel
-1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com)
+1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com). Be aware that the Release Notes (and all the App Record) has 2 localized versions: English UK (which containes NHS mentions) and another English (Australian by August/2019) for the other Territories. Paste the Release Notes on both.
```
Comment (j531) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: What does "App Record" mean?

Comment (j531) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: Some friendly suggestions. Lower case 'release notes' to match earlier use of the phrase.

```suggestion
1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com). Be aware that the release notes (and all the App Record) has 2 localized versions: English UK (which contains references to the NHS) and another English (Australian by August/2019) for the other territories. Paste the release notes on both. 
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 2]
```diff
@@ -101,11 +101,12 @@ The release process starts when the first build is provided to QA and ends when
 
 1. Automated QA effort (e.g. `5h`)
 2. Manual QA effort (e.g. `3h`)
-3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`)
+3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`). Consider only tickets that were raised during the release period, checking that its creation date is after the release branch cut.
```
Comment (j531) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: Some friendly suggestions.
```suggestion
3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`). Consider only tickets that were raised during the release period, checking that their creation dates were after the release branch cut. 
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 3]
```diff
@@ -101,11 +101,12 @@ The release process starts when the first build is provided to QA and ends when
...
-3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`)
+3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`). Consider only tickets that were raised during the release period, checking that its creation date is after the release branch cut. 
 4. Total effort
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.4.0 |Danilo Aliberti <br> Sergey Shulga | Automated: Parallel execution: 1 hour 28 minutes<br> Serial execution: 10 hours 15 minutes <br><br>Manual: 22 Hours 45 Minutes | `CNSMR-2521: 5days`, <br>`NRX-790: backend issue not resolved`, <br>`CNSMR-2515: 19 hours`, <br>`AV-910: 2days`, <br>`CNSMR-2509: 1day`, <br>`CNSMR-2506: 1day, 5hours`, <br>`CNSMR-2503: 1day, 1hour 10minutes`, <br>`CNSMR-2502: 6days and an hour`, <br>`CNSMR-2506: 1day, 5hours`| Total: **17days 31hours 10minutes** | 05.08.2019 | 12.08.2019 |
```
Comment (j531) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: 17 days and 31 hours? Doesn't that make 18 days and 7 hours? 😄

Comment (j531) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: Could you also update the style to match the other rows: `2d20h50min`? (And maybe update the previous two rows before this one where I used the incorrect style 🤦‍♂)?

**Review by j531 — COMMENTED** [review]
Few suggestions and comments.

File: Cookbook/Technical-Documents/ReleaseProcess.md (hunk 1 above)
Comment (daniloaliberti) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: It's the app content in the appStoreConnect. 
But actually, it's just about the release notes. Not all the app record. 👍

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ There are usually two release engineers working at any given time. It goes witho
 *It starts after all opened issues had been adressed and can take several cycles until QA's approval*
 
 1. Triger a new release build in the `#ios-build` channel
-1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com)
+1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com). Be aware that the release notes (and all the App Record) has 2 localized versions: English UK (which contains references to the NHS) and another English (Australian by August/2019) for the other territories. Paste the release notes on both.
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com). Be aware that the release notes has 2 localised versions: English UK (which contains references to the NHS) and another English (Australian by August/2019) for the other territories (the rest of the world). Paste the release notes on both if just one is provided in the #ios-launchpad.
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 5]
```diff
@@ -101,11 +101,12 @@ The release process starts when the first build is provided to QA and ends when
...
-3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`)
+3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`). Consider only tickets that were raised during the release period, checking that their creation dates were after the release branch cut. 
 4. Total effort
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.4.0 |Danilo Aliberti <br> Sergey Shulga | Automated: Parallel execution: 1 hour 28 minutes<br> Serial execution: 10 hours 15 minutes <br><br>Manual: 22 Hours 45 Minutes | `CNSMR-2521: 5days`, <br>`NRX-790: backend issue not resolved`, <br>`CNSMR-2515: 19 hours`, <br>`AV-910: 2days`, <br>`CNSMR-2509: 1day`, <br>`CNSMR-2506: 1day, 5hours`, <br>`CNSMR-2503: 1day, 1hour 10minutes`, <br>`CNSMR-2502: 6days and an hour`, <br>`CNSMR-2506: 1day, 5hours`| Total: **17days 31hours 10minutes** | 05.08.2019 | 12.08.2019 |
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.4.0 |Danilo Aliberti <br> Sergey Shulga | Automated: Parallel execution: 1 hour 28 minutes<br> Serial execution: 10 hours 15 minutes <br><br>Manual: 22 Hours 45 Minutes | `CNSMR-2521: 5days`, <br>`NRX-790: backend issue not resolved`, <br>`CNSMR-2515: 19 hours`, <br>`AV-910: 2days`, <br>`CNSMR-2509: 1day`, <br>`CNSMR-2506: 1day, 5hours`, <br>`CNSMR-2503: 1day, 1hour 10minutes`, <br>`CNSMR-2502: 6days and an hour`, <br>`CNSMR-2506: 1day, 5hours`| Total: **18days 7hours 10minutes** | 05.08.2019 | 12.08.2019 |
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 6]
```diff
@@ -101,11 +101,12 @@ The release process starts when the first build is provided to QA and ends when
...
-3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`)
+3. Delta between the jira ticket being open and marked as `done` or `wont fix`, for Engineering effort. (e.g. `UA-8289: 1h30`). Consider only tickets that were raised during the release period, checking that their creation dates were after the release branch cut. 
 4. Total effort
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.4.0 |Danilo Aliberti <br> Sergey Shulga | Automated: Parallel execution: 1 hour 28 minutes<br> Serial execution: 10 hours 15 minutes <br><br>Manual: 22 Hours 45 Minutes | `CNSMR-2521: 5days`, <br>`NRX-790: backend issue not resolved`, <br>`CNSMR-2515: 19 hours`, <br>`AV-910: 2days`, <br>`CNSMR-2509: 1day`, <br>`CNSMR-2506: 1day, 5hours`, <br>`CNSMR-2503: 1day, 1hour 10minutes`, <br>`CNSMR-2502: 6days and an hour`, <br>`CNSMR-2506: 1day, 5hours`| Total: **18days 7hours 10minutes** | 05.08.2019 | 12.08.2019 |
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.4.0 |Danilo Aliberti <br> Sergey Shulga | Automated: Parallel execution: 1 hour 28 minutes<br> Serial execution: 10 hours 15 minutes <br><br>Manual: 22 Hours 45 Minutes | `CNSMR-2521: 5days`, <br>`NRX-790: backend issue not resolved`, <br>`CNSMR-2515: 19 hours`, <br>`AV-910: 2days`, <br>`CNSMR-2509: 1day`, <br>`CNSMR-2506: 1day, 5hours`, <br>`CNSMR-2503: 1day, 1hour 10minutes`, <br>`CNSMR-2502: 6days and an hour`, <br>`CNSMR-2506: 1day, 5hours`| Total: **18d 7h 10m** | 05.08.2019 | 12.08.2019 |
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 7]
```diff
@@ -101,11 +101,12 @@ The release process starts when the first build is provided to QA and ends when
...
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 4.4.0 |Danilo Aliberti <br> Sergey Shulga | Automated: Parallel execution: 1 hour 28 minutes<br> Serial execution: 10 hours 15 minutes <br><br>Manual: 22 Hours 45 Minutes | `CNSMR-2521: 5days`, <br>`NRX-790: backend issue not resolved`, <br>`CNSMR-2515: 19 hours`, <br>`AV-910: 2days`, <br>`CNSMR-2509: 1day`, <br>`CNSMR-2506: 1day, 5hours`, <br>`CNSMR-2503: 1day, 1hour 10minutes`, <br>`CNSMR-2502: 6days and an hour`, <br>`CNSMR-2506: 1day, 5hours`| Total: **18d 7h 10m** | 05.08.2019 | 12.08.2019 |
 | 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated<br><br> Parallel execution: `1h 54min` Serial execution: `8h 41min`<br><br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: 1hr 30min`<br> `CNSMR-2338: 30min`<br> `CNSMR-2337: 60min`<br>  | Total: **44h 01min** | 22.07.2019 | 30.07.2019 |
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated<br><br> Parallel execution: `1h 54min` Serial execution: `8h 41min`<br><br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: 1hr 30min`<br> `CNSMR-2338: 30min`<br> `CNSMR-2337: 60min`<br>  | Total: **44h 01m** | 22.07.2019 | 30.07.2019 |
```

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 8]
```diff
@@ -101,12 +101,13 @@ The release process starts when the first build is provided to QA and ends when
...
+| 4.3.0  | Julien Ducret <br> Diego Petrucci | Automated<br><br> Parallel execution: `1h 54min` Serial execution: `8h 41min`<br><br>Manual: `24h 50min`| `CE-517: 30min`<br> `CNSMR-2143: 2days`<br> `CNSMR-2395: 0.5day`<br> `CNSMR-2333: 30min`<br> `CNSMR-2334: 30min`<br> `MON-4916: 30min`<br> `MON-4916: 30min`<br> `MON-4964: 30min`<br> `CE-512: 3hrs`<br> `CNSMR-2363: 1hr 30min`<br> `CNSMR-2338: 30min`<br> `CNSMR-2337: 60min`<br>  | Total: **44h 01m** | 22.07.2019 | 30.07.2019 |
 | 4.2.0  | Viorel Mihalache <br> Joshua Simmons | Automated: `8h 41min`<br>Manual: `24h 30min`| `AV-852: 3h`<br>`CNSMR-2173: 2h`<br>`NRX-724: 30min`<br>`CNSMR-2147: 8h`<br>`NCSMR-2167: 30min`<br>`NRX-720: 30min`<br>`CNSMR-2164: 30min`<br>`CNSMR-2162: 30min`<br>`AV-843: 30min`<br> | Total: **49h 11min** | 08.07.2019 | 15.07.2019 |
```
Comment (daniloaliberti) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
| 4.2.0  | Viorel Mihalache <br> Joshua Simmons | Automated: `8h 41min`<br>Manual: `24h 30min`| `AV-852: 3h`<br>`CNSMR-2173: 2h`<br>`NRX-724: 30min`<br>`CNSMR-2147: 8h`<br>`NCSMR-2167: 30min`<br>`NRX-720: 30min`<br>`CNSMR-2164: 30min`<br>`CNSMR-2162: 30min`<br>`AV-843: 30min`<br> | Total: **49h 11m** | 08.07.2019 | 15.07.2019 |
```

**Review by j531 — APPROVED** [review]

**Pull request merged**